test/box/gh-5928-several-listen-sockets.test.lua

```diff
@@ -0,0 +1,268 @@
+env = require('test_run')
+net_box = require('net.box')
+fio = require('fio')
+fiber = require('fiber')
+test_run = env.new()
+test_run:cmd("create server test with script='box/gh-5928-several-listen-sockets.lua'")
+
+test_run:cmd("setopt delimiter ';'")
+function str_split (inputstr, sep)
```

Comment:
Btw there's already str_split() func in load_cfg.lua
Why don't you take (inline) that implementation?

Reply:
str_split in load_cfg.lua is local function for `load_cfg`. I son't know how can i use it in test.